Torizon 7: Update Repo content

AllanKamimura opened the pull request:
README
- New torizon version 6.8.1 -> 7.1.0
- Unlist unsupported Demos

- Remove unsupported Demos

AllanKamimura committed:
torizon7:weston: Create test composes

To use the weston screenshot, we need to run the container with
the debug flag.
In this demo, we have a complex command, so it's hard to edit the
container command to include this flag.

At the same time, I don't want this variable on the deploy
composes. It should work even when this variable is not defined,
but it is going to give a warning.

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
torizon7: weston: Upgrade the Composes' version to Torizon-7

- Change the dockerhub image tag to 4.4.0
- To avoid code duplication, the new composes are defined by
the container name

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
torizon7:weston: update the Torizon Cloud metadata file

- Use the OS version as the package version

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
torizon7:weston: README: Add tested devices section

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
torizon7:weston: README: Update shield versions

- Torizon OS 6.8.0 -> 7.1.0
- Weston 3 -> 4.4.0

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
torizon7:qt: Create test composes

- Change the dockerhub Weston image tag to 4.4.0
- To avoid code duplication, the new composes are defined by
the container name instead of per device

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
torizon7: qt: Add a restart tag

- This is useful for deployed apps, since we don't want the user
to noticed that the app crashed
- Sometimes, the qt container tries to write draw on weston
before weston is ready, which leads to qt app crash

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
torizon7:qt: Upgrade Containers versions to Torizon 7

- Update the README information
- Update app.json Torizon Cloud metadata information
- Remove the Torizon 6 composes

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
README: update content

- New torizon version 6.8.1 -> 7.1.0
- Unlist unsupported Demos

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
Unlist unsupported Demos

Torizon 7: For now only Weston and Qt are going to be supported

Unlisted Demos from Torizon 6
- Chromium
- Codesys
- Node-red

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

leonheldattoradex commented:
@AllanKamimura I'm a bit confused about #18, #19 and this one so I'll just drop the comments here.

- make sure you're describing why the changes are being done; for example https://github.com/torizon/evaluation-apps/pull/19/files#diff-cda62f532dfb77ab29e2da013c37db97d417159c7f985a5297150f53e8a117eaL6 doesn't say why the image names are being changed and here we don't provide any reasons as to why these are unsupported https://github.com/torizon/evaluation-apps/pull/20/commits/31dd18f933a6bcca4d109ea7129e4424019ba7aa.
- I'm not sure I see value in having the "tested" devices section (https://github.com/torizon/evaluation-apps/pull/18/commits/719facc161f7b801db58b0776db41ecb2f61ee50#diff-39c9e6783282f54efff32c08da6912cd662789a24d208ea43ce03d87aaa6091eR54) if we're not synced with the Torizon OS releases. Ideally you would test these and report during our freeze period, which is a week before the actual real.
- when you have several MRs targetted to the same branch, please "merge-train" them, as in, target #18 into the main branch, #19 into #18 and #20 into #19, so it's imediately clear for me and for GitHub what's the merge sequence.

AllanKamimura committed:
torizon7:qt: Canonicalize docker compose

- This is required to use the lockbox feature in Torizon Cloud

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
torizon7:qt: Remove Tested Devices Section from README

Check PR#20 discussion
- There is no value in having test information

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura committed:
torizon7:weston: Remove Tested Devices Section from README

Check PR#20 discussion
- There is no value in having test information

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura commented:
Rebased against PR 
* Qt: Upgrade to Torizon-7 #19

AllanKamimura commented:
@leonheldattoradex asking for a re-review
- Improved commit messages in #19 
- Removed "tested devices" section from both #18 and #19 
- Not sure if this is what you meant, but I rebased the PRs to have an "obvious merge sequence"

AllanKamimura commented:
Closed on behalf of 
* https://github.com/torizon/evaluation-apps/pull/23
